macro/run_simScript.py

```diff
@@ -291,6 +300,28 @@
 # P8gen.SetMom(500.*u.GeV)
 # P8gen.SetId(-211)
  primGen.AddGenerator(P8gen)
+ 
+if simEngine == "Alpaca":#Rmin and Rmax of Decay Volume is needed...
+  #primGen.SetTarget(ship_geo.target.z0, 0.)
```

Comment:
Would suggest removing unnecessary commented lines of code. They are at best confusing.

Reply:
OK, let me do it.

Reply:
If you want to keep the comments for your local work, you can leave them in your local copy but remove them in the committed version (e.g. via `git add -p`).

Reply:
Perfect, thanks for the tip!

Reply:
`git add -p` is also great to selectively add some changes to make it easier to figure out which logically belong together (e.g. is you rename a variable just add the related changes). Maybe of interest to you: https://jeremykun.com/2020/01/14/the-communicative-value-of-using-git-well/

Reply:
Let me have a look.
Thank you for the tips again!